### PR TITLE
Document egg metadata

### PR DESCRIPTION
**Pull request by cournape**
Update documentation for the main classes of the library (`EggMetadata`, `EggBuilder`, `Platform` and `EPDPlatform`)

@sjagoe

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -246,6 +246,6 @@
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {'http://docs.python.org/': None}
 
-sys.path.append(os.path.abspath('.themes'))
-html_theme_path = ['.themes']
-html_theme = 'kr'
+import enthought_sphinx_theme
```
Comment (sjagoe) [inline doc/source/conf.py]: Where can I see this in action?

**Comment by sjagoe** [comment]
LGTM, as long as the enthought_sphinx_theme is good.

File: doc/source/conf.py (hunk 1 above)
Comment (cournape) [inline doc/source/conf.py]: Let me put it on okonomiyaki

Comment (cournape) [inline doc/source/conf.py]: http://okonomiyaki.readthedocs.org/en/document_egg_metadata/index.html

**Pull request merged**